content/Certification/Guides/PICSandPIXIT.md

```diff
@@ -0,0 +1,199 @@
+generation, just a helper to assist with this manual process. It is very
+important to go back and check that the values are as expected and to fill in
+the other PICS.
+
+### Verifying PICS using the IDM-10.4 certification test
+
+While not all PICS are verifiable on the device, we do have tests that verify
+the declared PICS against the device. This is one of the first tests that should
+be run at certification, as the PICS files are what determine which set of tests
+are required.
+
+To run these tests locally, follow the instructions at
+[Running Python tests locally](./python.md#running-tests-locally). The PICS
+checker test is TC-IDM-10.4, implemented in
+[TC_pics_checker.py](https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_pics_checker.py).
+These tests run a single set of PICS XML files against an endpoint. The
+`--endpoint` and `--PICS` flags are therefore required.
+
+Note that you can run tests locally against the PICS XMLs for an endpoint by
+supplying the name of the directory containing the set of PICS XML files for
+that endpoint.
+
+
+## Setting PIXITs for Matter devices
+
+Matter tests do not currently have support to read PIXIT values from the PICS
+XML files. Instead, tests implement PIXITs as test-specific flags. When running
+locally, these are specified on the command line. When running in the test
+harness, these are specified in the test parameters section of the test
+configuration.
+
+## PICS for test selection
+
+The official source that the CSA certification team uses to determine if all the
+required tests have been run at certification is the submitted set of PICS XML
+files and the PICS tool.
+
+To generate this set of tests for each endpoint, load all the filled PICS XML
+files for a single endpoint into the PICS tool and validate the PICS files. The
+PICS files should validate properly. This will also generate the list of test
+cases.
+
+Note that the test harness will also guide test selection by pre-selecting the
+required tests based on the PICS file, but it is the responsibility of the testers
+and the ATL to ensure that all the required tests are run and the results are
+submitted.
+
+[//]: # (## Creating PICS / PIXITs for new cluster and use in test plans)
+
+[//]: # (Placeholder for more information coming in a subsequent PR. Needs to cover)
+
+[//]: # (-   formatting - link out to test plans doc)
+[//]: # (-   requirement to have all the elements listed)
+[//]: # (-   how to do conformance in a way the PICS tool can handle)
+[//]: # (note - special attention to choice conformance, otherwise conformance)
+[//]: # (-   why you shouldn't use PICS values in tests other than at the top level)
+
+[//]: # (## Using PICS in test scripts)
+
+[//]: # (Cover top-level and step-wise pics in yaml and python and also where they should NOT be used.)
+
+[//]: # (## CI PICS format)
+
+[//]: # (CI PICS format, CI-only PICS and the gotchas there.)
```

Comment:
Do we want to leave this in the PR? Pardon my asciidoc naivete, but this will render into the handbook, no?

Reply:
The [//] suppresses them. md doesn't have a great comment format, but this is like cheater comments.